Sources/Core/Byte+Alphabet.swift

```diff
@@ -0,0 +1,178 @@
+
+// MARK: Alphabet
+
+extension Byte {
+    /// Returns whether or not the given byte is an arabic letter
+    public var isLetter: Bool {
+        return (.a ... .z).contains(self) || (.A ... .Z).contains(self)
```

Comment:
This can be done better. Here you're creating 2 buffers and searching through them.

`return (self >= .a && self <= .z) || (self >= .A && self <= .Z)` would be faster.